core/Common.php

```diff
@@ -188,7 +188,7 @@ public static function isRunningConsoleCommand()
      * @return string
      * @deprecated since 4.4 - directly use mb_substr instead
```

Comment:
Can we ditch this in 5.x since it's deprecated already? I know it's not a part of this particular PR, just wondering how long we will carry it around for.

Reply:
Normally the "process" would be to remove deprecations before the beta release. But as might be a breaking change for plugins we can't do it anymore. So we need to postpone that to Matomo 6.